tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -2798,6 +2798,27 @@ inline void Ceil(const RuntimeShape& input_shape, const float* input_data,
   }
 }
 
+inline float RoundToNearest(float value) {
```

Comment:
Add a comment that cfenv (for fesetround) is not yet supported universally on Android.